src/integrationtests/resources/features/ArtifactFlowIT.feature

```diff
@@ -0,0 +1,25 @@
+#Author: erik.edling@ericsson.com
+#Keywords Summary :
+#Feature: List of scenarios.
+#Scenario: Business rule through list of steps with arguments.
+#Given: Some precondition step
+#When: Some key actions
+#Then: To observe outcomes or validation
+#And,But: To enumerate more Given,When,Then steps
+#Scenario Outline: List of steps for data-driven as an Examples and <placeholder>
+#Examples: Container for s table
+#Background: List of steps run before each of the scenarios
+#""" (Doc Strings)
+#| (Data Tables)
+#@ (Tags/Labels):To group Scenarios
+#<> (placeholder)
+#""
+## (Comments)
+#Sample Feature Definition Template
+Feature: Test integration flow test
```

Comment:
Rename the feature to be more specific for Artifact flow, like the file is.

Reply:
There is also test 2 times in the name

Reply:
done